src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -235,6 +235,30 @@ internal override bool IsViewHit()
 			=> new Rect(default, RenderSize);
 #endif
 
+#if UNO_HAS_MANAGED_SCROLL_PRESENTER
```

Comment:
Why are you moving this in the shared file, while we already have a the `SCP.Managed.cs` file for this? If needed, we can rename the file `SCP.managed_pt.cs`

Reply:
No idea :). Moved to the `Managed` partial.